app/controllers/work_orders_controller.rb

```diff
@@ -5,17 +5,17 @@ class WorkOrdersController < ApplicationController
   before_action :work_order, only: [:show]
 
   def index
-    if session["user"]
-      @active_work_orders = WorkOrder.active.for_user(session["user"]["user"]["id"])
-      @pending_work_orders = WorkOrder.pending.for_user(session["user"]["user"]["id"])
+    if current_user
```

Comment:
Just found out there's actually a `user_signed_in?` which the Devise docs says you should use:

https://github.com/plataformatec/devise#controller-filters-and-helpers